lib/Models/prettyCoordinates.js

```diff
@@ -0,0 +1,43 @@
+import defined from 'terriajs-cesium/Source/Core/defined';
+import proj4 from 'proj4/lib/index.js';
+
+/**
+ * Turns the latitude / longitude in degrees into a human readable pretty strings.
+ */
+export function prettifyCoordinates(latitude, longitude, height, errorBar) {
+    var result = {};
+
+    result.latitude = Math.abs(latitude).toFixed(3) +
+        '°' + (latitude < 0.0 ? 'S' : 'N');
+    result.longitude = Math.abs(longitude).toFixed(3) +
+        '°' + (longitude < 0.0 ? 'W' : 'E');
+
+    if (defined(height)) {
+        result.elevation = Math.round(height) + (defined(errorBar) ? '±' +
+            Math.round(errorBar) : '') + 'm';
+    } else {
+        result.elevation = undefined;
+    }
+
+    return result;
+}
+
+/**
+ * Turns the latitude / longitude in degrees into a human readable pretty UTM zone representation.
+ */
+export function prettifyProjection(latitude, longitude, proj4Projection, proj4longlat, projectionUnits) {
```

Comment:
Probably better to put these in separate files.

Reply:
Done 558bda2.